packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -251,6 +288,15 @@ const NotificationDetail = () => {
     ).then(() => setPaymentLoading(false));
   }, [currentRecipient.payments]);
 
+  const handleReloadPayment = (noticeCode: string, creditorTaxId: string) => {
```

Comment:
I'ld update the fetchPaymentsInfo function to manage the case "all apyments" and "one payment" by passing as parameter an array of payments